queries.sql

```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+SELECT * from animals WHERE name LIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019.
+SELECT * FROM animals WHERE date_of_birth BETWEEN '01-01-2016' AND '01-01-2019';
```

Comment:
- [ ] Please update the constraint  for this query, because the animals born between 2016 - 2019 would start at the beginning of the year 2016 - end of the year 2019 ( inclusive )

Reply:
Thanks for your remark.